__tests__/app.test.js

```diff
@@ -22,6 +25,23 @@ describe("GET /not-a-path", () => {
   });
 });
 
+
+
+describe("GET /api", () => {
+    test("200: responds with an object describing all available endpoints", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body }) => {
+          for(const [key, value] of Object.entries(body.endpoints)){
+            expect(Object.keys(value)).toMatchObject(["description","queries","formatOfRequestBody","exampleResponse"])
```

Comment:
Could alternatively require in your endpoints.json directly (like you are in your controller) and expect response.body to be equal to it to make this more dynamic 🧠